graphics/partio/Portfile

```diff
@@ -30,3 +32,20 @@ depends_build-append \
                     port:doxygen \
                     port:swig-python
 depends_lib         port:zlib
+
+post-destroot {
+    copy ${worksrcpath}/src/data ${destroot}${prefix}/share/${name}/test
+    if {![file exists ${destroot}${prefix}/share/${name}/py]} {
```

Comment:
what's up with this?

Reply:
Oops, that stuff should be inside the `tests` variant. Let me fix that.

Reply:
You might want to instead add it to a documentation directory. Quite a lot of python ports do this for things like READMEs and LICENSEs, so there are loads of examples.

Here's one from [asciinema](https://github.com/macports/macports-ports/blob/0c80b0ba2f8b27ac6e6c72e1d8e7e5ee4aed5cad/sysutils/asciinema/Portfile):

```tcl
post-destroot {
    set docdir ${prefix}/share/doc/${subport}
    xinstall -d ${destroot}${docdir}
    xinstall -m 644 -W ${worksrcpath} \
        CHANGELOG.md \
        CONTRIBUTING.md \
        LICENSE \
        README.md \
        doc/asciicast-v1.md \
        doc/asciicast-v2.md \
        ${destroot}${docdir}
}
```

Reply:
There is already documentation getting generated into the documentation directory. And since the unit tests are already getting written to the shared folder, I just figured that it would be better to put the examples near the tests.

Reply:
Got it. Thanks 👍